README.md

```diff
@@ -4,13 +4,23 @@
 
 `atm-cli` is a command line tool for generating and working with MIDI files. It was purpose-built for
 All the Music, LLC to assist in its mission to enable musicians to make all of their music
-without the fear of frivolous copyright lawsuits. All code is released into the public domain
-via the [Creative Commons Attribute 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
+without the fear of frivolous copyright lawsuits. All code is freely available under the
+[Creative Commons Attribute 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
 If you're looking for a Rust library to generate and work with MIDI files, check out
 [the `libatm` project](https://github.com/allthemusicllc/libatm), on which this tool relies. For
 more information on All the Music, check out [allthemusic.info](http://allthemusic.info).  For more detailed
 information about the code, check out the crate documentation [here](https://allthemusicllc.github.io/atm-cli/atm/index.html).
 
+## Choice of License
+
+All datasets generated by All the Music, LLC have been released in the public domain, which means they are free for
+anyone to use for any purpose without restriction under copyright law. However, if we released the code into the
+public domain under CC0, we would have no control over entities using it with malicious intent. [CC4](http://creativecommons.org/licenses/by/4.0/)
```

Comment:
I chose this one

Reply:
> I chose this one

Creative Commons Attribution is never referred to as "CC4".
"CC0" is the name of the waiver. It's not "version 0". In fact, its name is "CC0 1.0".
"CC4" simply doens't exist. The abbreviated name is "CC BY". If you want to specify the number, it's "CC BY 4.0", a mere version of CC BY. There is no such thing as the CC4 license.